scripts/waitfor-bitbucket.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+set -ue
+
+# Starts a Bitbucket instance with a timebomb license (3 hours).
+# The instance is setup with an admin account (pw: admin) and an "ODSPIPELINETEST" project.
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ODS_PIPELINE_DIR=${SCRIPT_DIR%/*}
+
+INSECURE=""
+BITBUCKET_SERVER_HOST_PORT="7990"
+BITBUCKET_SERVER_CONTAINER_NAME="bitbucket-server-test"
+BITBUCKET_SERVER_IMAGE_NAME="atlassian/bitbucket"
+BITBUCKET_SERVER_IMAGE_TAG="7.6.5"
+BITBUCKET_POSTGRES_HOST_PORT="15432"
+BITBUCKET_POSTGRES_CONTAINER_NAME="bitbucket-postgres-test"
+BITBUCKET_POSTGRES_IMAGE_TAG="12"
+HELM_VALUES_FILE="${ODS_PIPELINE_DIR}/deploy/cd-namespace/chart/values.generated.yaml"
```

Comment:
Some of these are not used, right? Maybe we can run that through spellcheck?

Reply:
Yes adding shellcheck would make sense also. Perhaps in a separate issue?